src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const persons = document.querySelectorAll('ul li');
+
+const sorted = [...persons].sort((a, b) => {
+  return Number(b.dataset.salary
+    .slice(1)
+    .split(',')
+    .join(''))
+  - Number(a.dataset.salary
+    .slice(1)
+    .split(',')
+    .join(''));
```

Comment:
According to the terms of the task, you should create a separate function for converting the numbers.